### DISCUSSION TITLE
Does the latest RC support themes on WInUI ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [Live-Charts/LiveCharts2#1287] [Q&A]
`<PackageReference Include="LiveChartsCore.SkiaSharpView.Uno.WinUI" Version="2.0.0-rc1.2" />`

For example with dark theme the axis labels remain black therefore those cannot be read. 

I set this only in the XAML Page : 

`Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"`

**Comment by sigmarsson** [comment]
Are the axis labels reflecting to the current theme for you ? On Windows 11 here, labels become invisible along dark theme. - Thank you. 

`<PackageReference Include="LiveChartsCore.SkiaSharpView.Uno.WinUI" Version="2.0.0-rc2" />`

**Reply by frogcrush** [reply]
I'm encountering this issue too. I'm using Uno WinUI but that shouldn't make a difference.

**Comment by xD0135** [comment]
I have the same issue using the following version in WinUI
`<PackageReference Include="LiveChartsCore.SkiaSharpView.WinUI" Version="2.0.0-rc4.5" />`
